src/GafferScene/SceneAlgo.cpp

```diff
@@ -403,33 +404,25 @@ class CapturingMonitor : public Monitor
 		void processStarted( const Process *process ) override
 		{
 			const Plug *p = process->plug();
-			if( !p->parent<ScenePlug>() || p->getName() != m_scenePlugChildName )
+
+			CapturedProcess::Ptr capturedProcess( new CapturedProcess );
+			if( p->parent() && ( !p->parent<ScenePlug>() || p->getName() != m_scenePlugChildName ) )
```

Comment:
What's the extra `p->parent()` check about? I can't see anywhere we're vulnerable to it being null, and I'd have thought it's impossible to start a process for a parent-less plug anyway.